### PR TITLE
Save import on locahost

### PR DESCRIPTION
**Pull request by ba222ec**

### CODE REVIEW

File: public/js/buildingSaver.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		
+		/**
+		 * Fetches a building from localStorage.
+		 *
+		 * @param String id Name of the building to fetch
+		 * @return Object {"building":{"model":"qwerty","date":"1999-12-31"}} or false
+		 */
+		this.getBuilding = function(id) {
+			var building, all;
+			if (all = getAllBuildings()) {
```
Comment (davidsoderberg) [inline public/js/buildingSaver.js]: Is this if statment correct?

File: public/js/navigation.js [hunk 2]
```diff
@@ -92,43 +91,64 @@ jQuery(document).ready(function($) {
...
+			if (navigator.onLine) {	
+				$.ajax({
+					type: "POST",
+					url: requestUrl,
+					data: { name: name, model: dataString },
+					statusCode: {
+						201: function(result) {
+							// save in localStorage also
+							buildingSaver.saveBuildings(JSON.parse(result.data));
+							console.log(result);
```
Comment (davidsoderberg) [inline public/js/navigation.js]: Remove this ```console.log```?

Comment (ba222ec) [inline public/js/navigation.js]: '''console.log''' should be there until we have decided how to show right and error messages.

File: public/js/buildingSaver.js (hunk 1 above)
Comment (ba222ec) [inline public/js/buildingSaver.js]: It works but it's a bit unclear. I've made a new commit with *getAllbuildings()* moved to row 83.

**Pull request closed without merging**